py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -52,6 +52,7 @@ def aot_torch_tensorrt_aten_backend(
     )
 
 
+@fake_tensor_unsupported
```

Comment:
This is the ultimate backend everything goes through right? so does that mean we cant work on fake tensors? is this different than symbolic shapes?

Reply:
Also could we just turn a "fake_tensor" into an "ITensor" immediately. sounds like they are similar

Reply:
Yes this is the backend everything goes through, and my understanding is that `FakeTensors` are for use at compile time, and are distinct from `SymInt`s which are the symbolic shape representations.

The challenge with fake tensors right now is that _any_ tensors instantiated during the call are fake, which means that the constant tensors which we need to provide to TensorRT, as in the code snippet below, are "fake" and thus contain no value to be parsed. I think a long-term solution could be to support Fake Tensors fully, but this solution temporarily resolves TRT/Torch compatibility issues.

https://github.com/pytorch/TensorRT/blob/c7e79b2f3f48a1bd47266077f414117e42ba2ff5/py/torch_tensorrt/fx/converters/converter_utils.py#L247-L255

Reply:
Instantiated #1951 with a feature proposal and additional discussion